### ISSUE TITLE
Elements must meet minimum color contrast ratio thresholds

### ISSUE THREAD

**Issue by erinchambers** [CDRH/accessibility#15]
## Issue Description

Element has insufficient color contrast of 4.09 (foreground color: #6f767a, background color: #f5f1e7, font size: 11.2pt (14.9973px), font weight: normal). Expected contrast ratio of 4.5:1

``` html
<p id="site-description" class="site-description">Research, news, and features from the Center for Digital Research in the Humanities</p>
```

### Impact/Severity

Serious

### Link to relevant page(s)
 - [Header](https://cdrhdev.unl.edu/log/)

### Screenshot or Video

N/A

---

## Remediation Steps (if known)

Darken font color
